misc/python/materialize/feature_benchmark/benchmark_versioning.py

```diff
@@ -23,11 +23,11 @@
 
 # Consider increasing the scenario's class #version() if changes are expected to impact results!
 SHA256_BY_SCENARIO_FILE: dict[str, str] = {
-    "benchmark_main.py": "79c067f9a234a002f3264b38d983130fa93b15911dc559ce8690133e790416bc",
+    "benchmark_main.py": "1d61feccbf7178b6b430bbcbec00f70e008339d6f64f9cd9ea6208c13c1f5851",
```

Comment:
@def- does it cause any issues for the benchmarking that I had to update the sha256 here?

Reply:
Nope, totally fine. This is a mechanism we use to make people think about whether their change causes the benchmark results to become incomparable between changes. Changing comments doesn't matter.